library/Generator.js

```diff
@@ -33,7 +33,7 @@ class Generator {
 
   writeContentsJson(json, folderPath) {
     const path = folderPath.concat('/Contents.json')
-    Fs.writeFileSync(path, JSON.stringify(json))
+    Fs.writeFileSync(path, JSON.stringify(json, null, 2).replace(new RegExp(":", "g"), " :"))
```

Comment:
@mofneko Do you think it should be safer to use `replace(new RegExp(": ", "g"), " : ")` instead?

Reply:
Thank you for review

I think so too:]